### DISCUSSION TITLE
Starting today, I haven't been able to run "from whisper.utils import write_vtt" - any ideas why?

### DISCUSSION THREAD

**Discussion by runshouse** [openai/whisper#886] [Q&A] [answered]
The error I'm getting is:

ImportError: cannot import name 'write_vtt' from 'whisper.utils' (/usr/local/lib/python3.8/dist-packages/whisper/utils.py)

**Comment by runshouse** [comment, marked as answer]
Never mind I found the answer here https://github.com/openai/whisper/discussions/880

**Comment by sovdevs** [comment]
from whisper.utils import write_srt
ImportError: cannot import name 'write_srt' from 'whisper.utils' (/../python/lib/python3.11/site-packages/whisper/utils.py)

The suggested fix is to pip install just
openai-whisper==20230117
which installs its own whisper?

I am using
openai-whisper @ git+https://github.com/openai/whisper.git@b91c907694f96a3fb9da03d4bbdc83fbcd3a40a4
can import locally but not when deployed.

**Reply by glangford** [reply]
@sovdevs It looks like you are installing from the latest commit, and `write_srt` has been removed in a refactoring. See the discussion here

* https://github.com/openai/whisper/discussions/1289
